OpenRA.Mods.Common/Activities/Air/FlyAttack.cs

```diff
@@ -271,7 +278,10 @@ protected override void OnFirstRun(Actor self)
 			if (target.IsValidFor(self))
 			{
 				QueueChild(new Fly(self, target, target.CenterPosition));
-				QueueChild(new Fly(self, target, exitRange, WDist.MaxValue, target.CenterPosition));
+				QueueChild(new FlyForward(self, exitRange));
+
+				// Turn towards the target for another run
+				QueueChild(new Fly(self, target, exitRange + new WDist(aircraft.Info.Speed * 256 / aircraft.Info.TurnSpeed.Angle), WDist.MaxValue, target.CenterPosition));
```

Comment:
Can we have a comment explaining the meaning of `aircraft.Info.Speed * 256 / aircraft.Info.TurnSpeed.Angle`?